ipn/ipnlocal/local.go

```diff
@@ -393,6 +393,15 @@ type metrics struct {
 	// advertisedRoutes is a metric that counts the number of network routes that are advertised by the local node.
 	// This informs the user of how many routes are being advertised by the local node, excluding exit routes.
 	advertisedRoutes *usermetric.Gauge
+
+	// approvedRoutes is a metric that counts the number of network routes served by the local node and approved
```

Comment:
nit: could you please change "counts" to "reports" here and in other 2 comments? Just to make sure nobody thinks it's a counter rather than a gauge.

Reply:
done